README.md

```diff
@@ -8,8 +8,8 @@ This repository is intended to setup a software workflow for using the EIC smear
 This has only been tested on RCF, where access to the EIC environment is present. In order to use this, it is expected you have setup this environment via
 
 ```
-$ setenv EIC_LEVEL pro 
-$ source /afs/rhic.bnl.gov/eic/restructured/etc/eic_cshrc.csh
+setenv EIC_LEVEL pro 
```

Comment:
Please leave the `$` signs, this is standard for indicating "at the command line"

Reply:
Will do -- as a heads up I may have gotten rid of them in the line with the example command to run the python file as well. I will fix this.